### DISCUSSION TITLE
A question about the 55 extra characters in "what I'm working on"

### DISCUSSION THREAD

**Discussion by Endermaster3690** [ScratchAddons/ScratchAddons#6918] [Help] [answered]
why can't it be more? why can't we also add this in "about me"?

My guess is that it's something that ST forgot to fix and left the hard limit of what I'm working on at 255

**Comment by CST1229** [comment, marked as answer]
> My guess is that it's something that ST forgot to fix and left the hard limit of what I'm working on at 255

it is
